artifactory/commands/container/strategies/runnativestrategy.go

```diff
@@ -0,0 +1,115 @@
+package strategies
+
+import (
+	"github.com/jfrog/jfrog-cli-artifactory/artifactory/commands/container/dockerfileutils"
+	container "github.com/jfrog/jfrog-cli-artifactory/artifactory/commands/ocicontainer"
+	"github.com/jfrog/jfrog-cli-core/v2/artifactory/utils"
+	"github.com/jfrog/jfrog-cli-core/v2/common/build"
+	"github.com/jfrog/jfrog-client-go/utils/errorutils"
+	"github.com/jfrog/jfrog-client-go/utils/log"
+)
+
+// RunNativeStrategy runs docker build directly without JFrog enhancements
+type RunNativeStrategy struct {
+	DockerBuildStrategyBase
+}
+
+func NewRunNativeStrategy(options DockerBuildOptions) *RunNativeStrategy {
+	return &RunNativeStrategy{
+		DockerBuildStrategyBase: DockerBuildStrategyBase{
+			containerManager:   container.NewManager(container.DockerClient),
+			dockerBuildOptions: options,
+		},
+	}
+}
+
+func (s *RunNativeStrategy) Execute(cmdParams []string, buildConfig *build.BuildConfiguration) error {
+	log.Info("Running docker build in native mode (JFROG_RUN_NATIVE=true)")
+
+	// Run native docker build directly
+	err := s.GetContainerManager().RunNativeCmd(cmdParams)
+	if err != nil {
+		return err
+	}
+
+	// Check if build-info collection is needed using existing BuildConfiguration method
+	toCollect, err := buildConfig.IsCollectBuildInfo()
+	if err != nil {
+		return err
+	}
+	if toCollect {
+		// Collect build-info for docker build
+		err = s.collectBuildInfo(cmdParams, buildConfig)
+		if err != nil {
+			// just warn, no need to fail the build if build info collection fails
+			log.Warn("Failed to collect build info. Error:", err)
+		}
+	}
+	return nil
+}
+
+func (s *RunNativeStrategy) collectBuildInfo(cmdParams []string, buildConfig *build.BuildConfiguration) error {
+	log.Info("Collecting build info...")
+	if s.dockerBuildOptions.ImageTag == "" {
+		log.Warn("Could not extract image tag from build command")
+		return nil
+	}
+
+	// Parse Dockerfile to get base images
+	dockerfilePath := s.dockerBuildOptions.DockerFilePath
+	baseImageInfos, err := dockerfileutils.ParseDockerfileBaseImages(dockerfilePath)
+	if err != nil {
+		return errorutils.CheckErrorf("Failed to parse Dockerfile: %s", err.Error())
+	}
+
+	if len(baseImageInfos) == 0 {
+		log.Info("No base images found in Dockerfile")
+		return nil
+	}
+
+	// Use server details from the strategy (set by BuildCommand)
+	serverDetails := s.GetServerDetails()
```

Comment:
may be nil check for serverDetails is good idea since setServerDetails allows nil